server/src/main/java/org/apache/iotdb/db/engine/storagegroup/StorageGroupProcessor.java

```diff
@@ -462,7 +482,7 @@ private void insertBatchToTsFileProcessor(BatchInsertPlan batchInsertPlan,
       List<Integer> indexes, boolean sequence, Integer[] results, long timeRange)
       throws QueryProcessException {
 
-    TsFileProcessor tsFileProcessor = getOrCreateTsFileProcessor(fromTimeRangeToTime(timeRange),
+    TsFileProcessor tsFileProcessor = getOrCreateTsFileProcessor(timeRange,
```

Comment:
It would be better to change to "timeRangeId" as a single number cannot be called range. Fix elsewhere, please.

Reply:
fixed~